src/ng-generate/components/form/generators/components/fields/either/files/__name@dasherize__.component.html.template

```diff
@@ -1,5 +1,5 @@
 <form [formGroup]="formGroup">
-    <fieldset class="form-group">
+    <fieldset class="form-group" data-test="<%= options.fieldConfig.name %>">
```

Comment:
I think for compound fields it would be more convenient to add its type as well, `<%= options.fieldConfig.name %>-either` in this case.

Reply:
done